tools/accuracy_checker/configs/unet-camvid-onnx-0001.yml

```diff
@@ -0,0 +1,55 @@
+models:
+  - name: unet-camvid-onnx-0001
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/unet-camvid-onnx-0001/FP32/unet-camvid-onnx-0001.xml
+        weights: intel/unet-camvid-onnx-0001/FP32/unet-camvid-onnx-0001.bin
+        adapter: segmentation
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   intel/unet-camvid-onnx-0001/FP16/unet-camvid-onnx-0001.xml
+        weights: intel/unet-camvid-onnx-0001/FP16/unet-camvid-onnx-0001.bin
+        adapter: segmentation
+
+      - framework: dlsdk
+        tags:
+          - FP32-INT8
+        model:   intel/unet-camvid-onnx-0001/FP32-INT8/unet-camvid-onnx-0001.xml
+        weights: intel/unet-camvid-onnx-0001/FP32-INT8/unet-camvid-onnx-0001.bin
+        adapter: segmentation
+
+      - framework: dlsdk
+        tags:
+          - FP16-INT8
+        model:   intel/unet-camvid-onnx-0001/FP16-INT8/unet-camvid-onnx-0001.xml
+        weights: intel/unet-camvid-onnx-0001/FP16-INT8/unet-camvid-onnx-0001.bin
+        adapter: segmentation
+
+    datasets:
+      - name: CamVid
+
+        preprocessing:
+          - type: resize
+            dst_height: 368
+            dst_width: 480
+            use_pillow: True
+            interpolation: BILINEAR
+          - type: bgr_to_rgb
```

Comment:
why bgr_to_rgb used? There is no way to convert model in BGR?

Reply:
if original model trained with pillow usage, than you need:
```yml
reader: pillow_imread
preprocessing:
         -  type: resize
            dst_height: 368
            dst_width: 480
            use_pillow: True
            interpolation: BILINEAR
         -  type: rgb_to_bgr

Reply:
this model was a candidate for Open Model Zoo about year ago. Re bgr_to_rgb it is a good question, need to ask @AlexKoff88 on details.

Reply:
my old config has reading in PIL, so the configuration which I described abowe is correct
https://github.com/opencv/open_model_zoo/pull/694

Probably bgr_to_rgb was used because AC has not support of rgb_to_bgr preprocessor (they makes the same changes, just name mislead)

Reply:
@vshampor, could you please comment on that?